src/main/java/backend/controller/PostingController.kt

```diff
@@ -67,20 +67,26 @@ class PostingController(private val postingService: PostingService,
         val posting = postingService.createPosting(user, body.text, body.uploadMediaTypes, locationCoord, clientDate)
         posting.media.forEach { it.uploadToken = getSignedJwtToken(JWT_SECRET, it.id.toString()) }
 
-        return PostingView(posting, null)
+        return PostingView(posting, null, user.account.id)
```

Comment:
In the long run I think we should think about whether or not we want to keep using this pattern or do something cleaner here